tools/interactive/interactivetool_metashark.xml

```diff
@@ -0,0 +1,58 @@
+<tool id="interactive_tool_metashark" tool_type="interactive" name="MetaShARK" version="@TOOL_VERSION@">
+    <description>Metadata Shiny Automated Resource and Knowledge</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.7.0</token>
+    </macros>
+    <requirements>
+        <container type="docker">eliearnaud/metashark:@TOOL_VERSION@</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="MetaShARK visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/</url>
+        </entry_point>
+    </entry_points>
+    <command><![CDATA[
+
+mkdir -p /srv/shiny-server/data/ &&
+cp '$infile' /srv/shiny-server/data/inputdata.txt &&
```

Comment:
@yvanlebras you are copying here only one file, but you can put in multiple files in the UI

Reply:
True! Thank you, I was updating the param to allow multiple files...and forget to change the command line!

Reply:
I tried to add something looking at https://github.com/yvanlebras/galaxy/blob/dev/tools/interactive/interactivetool_jupyter_notebook.xml#L24 . Can you say to me if this is good, and even better to use ln -sf than a cp ?